l1-l2-deposit-withdrawal/scripts/example.js

```diff
@@ -63,16 +63,21 @@ async function main() {
     'L1 ERC20', //name
   )
   await L1_ERC20.deployTransaction.wait()
+  console.log(`   @addr ${L1_ERC20.address}`)
```

Comment:
The added console.logs are just outputting the addresses but maybe they should be prefixed with some wording on what address if printed, e.g. "L1_ERC20 address" and "L2_ERC20 address" instead of just "0x..." outputs